specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-08-01/examples/virtualMachineExamples/VirtualMachine_Get_VM_With_InstanceView_WithDiskControllerType.json

```diff
@@ -0,0 +1,219 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "myResourceGroup",
+    "api-version": "2022-08-01",
+    "vmName": "myVM"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "name": "myVirtualMachineScaleSet",
+        "id": "/subscriptions/{subscription-id}/resourceGroups/myResourceGroup/providers/Microsoft.Compute/virtualMachineScaleSets/myVirtualMachineScaleSet",
+        "type": "Microsoft.Compute/virtualMachineScaleSets",
```

Comment:
we are adding VMSS resource examples for Get VM with instance view response, pls. update the sample for VM resource.

Reply:
corrected